tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -54,7 +54,7 @@
     <!-- Integration: GraphQL -->
     <!-- Assembly: GraphQL -->
     <!-- Latest package https://www.nuget.org/packages/GraphQL/7.3.0 -->
-    <PackageReference Include="GraphQL" Version="7.3.0" />
+    <PackageReference Include="GraphQL" Version="5.4.0" />
```

Comment:
wait what?

Reply:
Re-ran locally and didn't see the issue 🤷‍♂️ Maybe some wires getting crossed with it generating the PR before we merged Graphql 7. It's ok now anyway